scrapy_poet/utils/_deprecation.py

```diff
@@ -0,0 +1,110 @@
+import inspect
+from warnings import warn
+
+
+def _clspath(cls, forced=None):
+    if forced is not None:
+        return forced
+    return f"{cls.__module__}.{cls.__name__}"
+
+
+def _create_deprecated_class(
```

Comment:
I think it might be better to use the same function either from scrapy or from web-poet, instead of copying it here. Scrapy's might be a bit better, because it's public.

Reply:
Good call! Removed this. https://github.com/scrapinghub/scrapy-poet/pull/103/commits/190e3a67aebff3be29440317b19533a78b98791b